app/components/billing-information.js

```diff
@@ -0,0 +1,56 @@
+import Component from '@ember/component';
+import { getCountries } from '../utils/countries';
+import { computed } from '@ember/object';
+import { not } from '@ember/object/computed';
+
+export default Component.extend({
+  plans: null,
+  selectedPlan: null,
+  showAnnual: false,
+  countries: getCountries(),
+  plansToShow: ['Bootstrap', 'Startup', 'Small Business', 'Premium'],
+  defaultPlan: 'Startup',
+  showMonthly: not('showAnnual'),
+
+  monthlyPlans: computed('plans', function () {
```

Comment:
When computed property depends not only on value, but also on value contents, the dependent key should contain the entire path, e.g. `plan.name`, `plan.annual`, etc. If a property is array (or has array somewhere in the path), it can be marked for tracking by adding `.[]` or `.@each`, which will make computed property to observe array content. In this case, the path would be `plans.@each`. This path will tell Ember to update computed property whenever the contents of `plans` array changes (items added or removed). If the property needs to update when certain fields of array items change, it also needs to be reflected in the path: `plans.@each.annual`, `plans.@each.name`, etc. There is a limitation in this structure - array items can be observed only for 1 level deep, e.g `plans.@each.builds.name` is invalid path. As a shortcut, it's possible to combine dependent keys for  deep paths - instead of `plans.@each.name`, `plans.@each.annual` you can just write `plans.@each.{name,annual}`.

Because of incorrect dependent key, the property does not update correctly. Basically, currently it updates only when the `plans` property itself is set, it doesn't track the contents.